lib/shipit/github_app.rb

```diff
@@ -125,7 +125,7 @@ def enterprise?
     end
 
     def new_client(options = {})
-      options.reverse_merge(api_endpoint: api_endpoint) if api_endpoint
+      options.reverse_merge!(api_endpoint: api_endpoint) if api_endpoint
```

Comment:
`options = options.reverse_merge` would be preferable, as to avoid mutating the argument.